{"name": "Проверка авторизации с неверным логином и паролем", "status": "passed", "description": "Регистрируем пользователя, затем авторизуемся с неверным логином и паролем. Проверяем, что юзер не авторизовался и текст ошибки: \"email or password are incorrect\". После проверки удаляем юзера", "steps": [{"name": "Создаём юзера", "status": "passed", "start": 1717572712012, "stop": 1717572712301}, {"name": "Авторизуем юзера", "status": "passed", "start": 1717572712301, "stop": 1717572713328}, {"name": "Удаляем юзера", "status": "passed", "start": 1717572713328, "stop": 1717572713767}], "start": 1717572712012, "stop": 1717572713767, "uuid": "74ab88be-7a25-4efa-a57d-4d9785277e64", "historyId": "e0688489092ad09f221afeedb9145b44", "testCaseId": "e0688489092ad09f221afeedb9145b44", "fullName": "tests.test_auth_user.TestAuthUser#test_auth_with_incorrect_auth_body", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestAuthUser"}, {"name": "host", "value": "Kotiki"}, {"name": "thread", "value": "15004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}